src/ontology/mondo-edit.obo

```diff
@@ -127660,6 +127660,7 @@ xref: MESH:C563239 {source="MONDO:equivalentTo"}
 xref: OMIM:105805 {source="Orphanet:99092", source="MONDO:equivalentTo", source="Orphanet:99092/e"}
 xref: Orphanet:99092 {source="MONDO:equivalentTo", source="OMIM:105805"}
 xref: UMLS:C1387721 {source="Orphanet:99092", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:105805"}
+is_a: MONDO:0004995 {source="https://orcid.org/0000-0001-6718-3559", source="https://orcid.org/0000-0001-9310-0163"} ! cardiovascular disorder
 is_a: MONDO:0019512 {source="https://orcid.org/0000-0002-3458-4839"} ! congenital heart malformation
 is_a: MONDO:0021147 {source="https://orcid.org/0000-0002-4142-7153"} ! disorder of development or morphogenesis
 property_value: confidence "7.125" xsd:double
```

Comment:
see below- the GitHub URL has already been added, so it's not being added with the [ROBOT template](https://docs.google.com/spreadsheets/d/1fIqp9B434HjNWZmjqWKYXjvDY6e3LsTg/edit#gid=609591374)